t/fits.t

```diff
@@ -316,6 +316,15 @@ SKIP:{
 	ok(all($b==$a),"The new image matches the old one (longlong)");
 	unlink $file;
 }
-	
+
+###############################
+# Check that tilde expansion works
+my $tildefile = cfile('~',"PDL-IO-FITS-test_$$.fits");
+eval { sequence(3,5,2)->wfits($tildefile); };
+ok(!$@, "wfits tilde expansion didn't fail");
```

Comment:
We're also using `Test::Exception`, so that could even be

```perl
use Test::Exception;
lives_ok {
    sequence(3,5,2)->wfits($tildefile)
} "wfits tilde expansion didn't fail";
```